src/js/components/Diagram/stories/Progressing.js

```diff
@@ -39,7 +39,7 @@ export const Progressing = () => {
   useEffect(() => {
     const timer = setInterval(() => {
       dispatch();
-    }, 2000);
+    }, 3000);
```

Comment:
Is there a reason we had to increase this 1s?

Reply:
For this story as a whole, I wanted to extend the duration of the animations to show a smoother transit. Increasing the timer here allows for longer durations on animations to better showcase to users the animations available on Diagram.